CHANGELOG.md

```diff
@@ -0,0 +1,41 @@
+# Changelog
+All notable eosc-kc changes of Keycloak will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+For Keycloak upstream changelog please see https://www.keycloak.org/docs/latest/release_notes/index.html.
+Full Keycloak upstream jira issue can be shown if filtered by Fix version. For example [Keycloak jira issue for 15.0.2 version](https://issues.redhat.com/browse/KEYCLOAK-19161?jql=project%20%3D%20keycloak%20and%20fixVersion%20%3D%2015.0.2)
+
+## [v15.0.2-r1.0.4] - 4-11-2021
+### Added
+- Attribute IdP mapper taking into account emailVerified User field. [EOSC-KC-70](https://github.com/eosc-kc/keycloak/issues/70)
+
+### Fixed
+- Fix a bug for Identity Providers in ModelToRepresentation
+
+## [v15.0.2-r1.0.3] - 18-10-2021
+### Added
+- new release created on tag
+
+## [v15.0.2-r1.0.2] - 5-10-2021
+### Fixed
+- Brokered SAML logins fail due to wrong InResponseTo in 15.x. [KEYCLOAK-19143](https://issues.redhat.com/browse/KEYCLOAK-19143)
+
+## [v15.0.2-r1.0.1] - 29-9-2021
+### Added
+- Support for SAML IdP federation
+- User reaccepting terms and conditions. [EOSC-KC-48](https://github.com/eosc-kc/keycloak/issues/48)
```

Comment:
Please add these as well:

Create a minimal realm representation and use it in Admin Console [EOSC-KC-93](https://github.com/eosc-kc/keycloak/issues/93)
Allow fetching of Identity Providers in pages and enhance Admin Console by using that [EOSC-KC-73](https://github.com/keycloak/keycloak/discussions/8608)

Reply:
I have previously added EOSC-KC-49 with description 'Terms and Conditions - periodically ask for reacceptance.' . I have changed description based on your comment. However, I believe that I should use term 'Terms and Conditions' that used in Keycloak documentation.

See my add about EOSC-KC-73.